api/swagger.yaml

```diff
@@ -2683,6 +2683,13 @@ definitions:
               - "default"
               - "process"
               - "hyperv"
+      NetworkAttachmentSpec:
+        description: "Read-only spec type for non-swarm containers attached to swarm overlay networks"
```

Comment:
Think we can use a common description for these being mutually exclusive (let me add a commit to address that)